src/server/server/gator.py

```diff
@@ -14,10 +14,18 @@ def is_dev_env_server():
 DEBUG_MODE = is_dev_env_server()
 
 ROUTING = [
-    ('/', handlers.LoginPageHandler),
-    ('/home', handlers.HomePageHandler)
+    ('/login', handlers.LoginHandler),
+    ('/register', handlers.RegisterHandler),
+    ('/disconnect', handlers.DisconnectHandler),
+    ('/link/(click_link|up_vote|down_vote)/(.*)', handlers.LinkHandler),
+    ('/', handlers.HomeHandler),
 ]
 
+CONFIGURATION = {}
+CONFIGURATION['webapp2_extras.sessions'] = {
+    'secret_key': 'maybe_we_should_generate_a_random_key',
+}
```

Comment:
I guess to have this key here is temporary, maybe you could create a new github issue with roughly what need to be done Or just a link of what is the way ?